packages/db/src/pouch/index.ts

```diff
@@ -0,0 +1,72 @@
+import path from "path";
+
+export * from "./types";
+
+import { DatabasesOptions } from "./databases";
+
+import { FSDatabases } from "./fs";
+import { MemoryDatabases } from "./memory";
+import { SqliteDatabases } from "./sqlite";
+
+import { Collections } from "@truffle/db/meta";
+import { Databases, Definitions } from "./types";
+
+export interface WorkspaceConfig {
```

Comment:
Do we still want to call this `WorkspaceConfig`?

Reply:
Nope, probably not! I'll change this; shouldn't be hard to come up with a new name.